test/runnable/test_cdvecfill.d

```diff
@@ -87,39 +87,883 @@ version (update)
             auto content = src.readText;
             auto f = File(src, "w");
             auto orng = f.lockingTextWriter;
-            replaceFirstInto!((_, orng) => formattedWrite(orng, "// dfmt off\n%s// dfmt on", sink.data))(orng, content, ctRegex!(`^// dfmt off[^$]*// dfmt on$`, "m"));
+            version(D_PIC)
+            {
+                replaceFirstInto!((_, orng) => formattedWrite(orng, "// dfmtpic off\n%s// dfmtpic on", sink.data))(orng, content, ctRegex!(`^// dfmtpic off[^$]*// dfmtpic on$`, "m"));
```

Comment:
Ehm I think you need to leave `// dfmt off` as it's a known directive.
Maybe sth. like:

```
// PIC\n// dfmt off[^$]*// dfmt off
```

See also: https://github.com/dlang-community/dfmt#disabling-formatting

Reply:
😆 I thought that was a strange moniker to use.  I've updated to code to keep the `// dfmt` directives and use different strings for the codegen.